### PR TITLE
CR and trailing space

### PR DESCRIPTION
**Pull request by johnyf**
These changes remove trailing ` `, `\r`, and replace literal `\r` characters with the escape sequence `\r`.

### CODE REVIEW

File: test/Makefile [hunk 1]
```diff
@@ -56,7 +56,7 @@ coqscript: basics.vo
                       >tmp_test.v \
 	     && (echo $$i '-> coqc' | tr -d '\n'; \
 	         coqc -I .. tmp_test.v); then \
-	    echo '' | tr -d '\n'; \
+	    echo $$'\r' | tr -d '\n'; \
```
Comment (damiendoligez) [inline test/Makefile]: In modern Unix, I'd write it like this:
```suggestion
	    printf '\r'; \
```

File: test/Makefile [hunk 2]
```diff
@@ -75,7 +75,7 @@ coqterm: basics.vo
...
```
Comment (damiendoligez) [inline test/Makefile]:
```suggestion
	    printf '\r'; \
```

**Review by damiendoligez — CHANGES_REQUESTED** [review]
Just a small suggestion. Otherwise, it looks good.

File: test/Makefile (hunk 1 above)
Comment (johnyf) [reply test/Makefile]: Thank you for suggesting these changes, I will rebase to update these lines.

**Comment by johnyf** [comment]
Thank you for reviewing these changes, and suggesting changes, I applied the suggestions and rebased the branch.

**Review by damiendoligez — APPROVED** [review]

**Comment by damiendoligez** [comment]
Thanks for the PR.

**Comment by johnyf** [comment]
Thank you for approving and merging these changes.

**Pull request merged**